File: 03.business-os/NameCardManager/android/app/src/main/java/com/lsam/NameCardManager/wrapper/BusinessOSCommonOSWrapperRegistry.java
package com.lsam.NameCardManager.wrapper;

public final class BusinessOSCommonOSWrapperRegistry {

    public BusinessOSCommonOSWrapperDescriptor createDefault() {
        return new BusinessOSCommonOSWrapperDescriptor(
                "NameCardManager",
                "~/03.civilization-development/12.common-os",
                "~/03.civilization-development/03.business-os/_commonos",
                "shared_provider",
                "os_side_consumer",
                "adapter",
                "bridge",
                "mapper",
                "presenter",
                "theme",
                "sync",
                "CommonOS",
                "NameCardManager"
        );
    }
}

File: 03.business-os/NameCardManager/iphone/NameCardManager/BusinessOSCommonOSWrapper/BusinessOSCommonOSWrapperRegistry.swift
import Foundation

enum BusinessOSCommonOSWrapperRegistry {
    static func createDefault() -> BusinessOSCommonOSWrapperDescriptor {
        BusinessOSCommonOSWrapperDescriptor(
            appName: "NameCardManager",
            providerRoot: "~/03.civilization-development/12.common-os",
            consumerRoot: "~/03.civilization-development/03.business-os/_commonos",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            adapterModule: "adapter",
            bridgeModule: "bridge",
            mapperModule: "mapper",
            presenterModule: "presenter",
            themeModule: "theme",
            syncModule: "sync",
            uiOwner: "CommonOS",
            businessOwner: "NameCardManager"
        )
    }
}
